subsystem/cluster_test.go

```diff
@@ -735,7 +735,7 @@ var _ = Describe("V2ListClusters", func() {
 			NewClusterParams: &models.ClusterCreateParams{
 				BaseDNSDomain:     "example.com",
 				Name:              swag.String("test-cluster"),
-				OpenshiftVersion:  swag.String(openshiftVersion),
```

Comment:
code change is a mistake?

Reply:
No, now that 4.15 is the default, all subsystem tests that use SDN and/or VIP auto allocation fail is it is not supported anymore in 4.15